modules/async-icq/testing/demo-simapp/app/simulation_test.go

```diff
@@ -0,0 +1,113 @@
+package app_test
+
+import (
+	"os"
+	"testing"
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/baseapp"
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/simapp"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	simulationtypes "github.com/cosmos/cosmos-sdk/types/simulation"
+	"github.com/cosmos/cosmos-sdk/x/simulation"
+	"github.com/quasar-finance/interchain-query-demo/app"
```

Comment:
should be updated to reference the `/app` package in this same repo vs the quasar repo